content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -230,6 +247,68 @@ dual-stack.)
      selector:
        app: MyApp
    ```
+=======
+```shell
+kubectl get svc my-service -o yaml
+```
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: MyApp
+  name: my-service
+spec:
+  clusterIP: 10.0.197.123
+  clusterIPs:
+  - 10.0.197.123
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app.kubernetes.io/name: MyApp
+  type: ClusterIP
+status:
+  loadBalancer: {}
+```
+
+1. When dual-stack is enabled on a cluster, existing [headless Services](/docs/concepts/services-networking/service/#headless-services) with selectors are configured by the control plane to set `.spec.ipFamilyPolicy` to `SingleStack` and set `.spec.ipFamilies` to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-apiserver) even though `.spec.ClusterIP` is set to `None`.
+
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
+
+   You can validate this behavior by using kubectl to inspect an existing headless service with selectors.
+
+```shell
+kubectl get svc my-service -o yaml
+```
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: MyApp
+  name: my-service
+spec:
+  clusterIP: None
+  clusterIPs:
+  - None
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app.kubernetes.io/name: MyApp
+```
+>>>>>>> cca4c78633 ([en] update en docs to use recommended labels)
```

Comment:
This looks like a manual merge gone wrong; please fix it.

Reply:
Apologies, I must have missed this during one of the rebases I had to do.